src/Home/ExposureDetectionStatus.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FunctionComponent } from "react"
+import { ScrollView, StyleSheet } from "react-native"
+import { useTranslation } from "react-i18next"
+
+import { Text } from "../components"
+import { useStatusBarEffect } from "../navigation"
+
+import BluetoothActivationStatus from "./BluetoothActivationStatus"
+import ExposureNotificationsActivationStatus from "./ExposureNotificationsActivationStatus"
+import LocationActivationStatus from "./LocationActivationStatus"
+import { useExposureDetectionStatus } from "./useExposureDetectionStatus"
+
+import { Colors, Spacing, Typography } from "../styles"
+
+const ExposureDetectionStatus: FunctionComponent = () => {
+  useStatusBarEffect("dark-content", Colors.primaryLightBackground)
+  const { t } = useTranslation()
+  const { exposureDetectionStatus } = useExposureDetectionStatus()
+
+  const subheaderText = exposureDetectionStatus
+    ? t("exposure_scanning_status.your_device_is_scanning")
+    : t("exposure_scanning_status.your_device_is_not")
+
+  return (
+    <ScrollView
+      style={style.container}
+      contentContainerStyle={style.contentContainer}
+    >
+      <Text style={style.headerText}>
+        {t("screen_titles.exposure_scanning")}
+      </Text>
+      <Text style={style.subheaderText}>{subheaderText}</Text>
+      <BluetoothActivationStatus />
+      <ExposureNotificationsActivationStatus />
+      <LocationActivationStatus />
```

Comment:
What do you think about moving the logic that determines whether or not to show this component into its parent, i.e. `ExposureDetectionStatus` ?

Reply:
👍